### PR TITLE
Improvements on Yao's millionaires example

### PR DESCRIPTION
**Pull request by grydz**

### CODE REVIEW

File: examples/yaos_millionaires/README.md [hunk 1]
```diff
@@ -13,14 +41,82 @@ $ cenclave localtest --code src/ \
                      --test tests/
 ```
 
-## Create Cosmian Enclave package with the code and the container image
+### Create a package for Cosmian Enclave
```
Comment (Manuthor) [inline examples/yaos_millionaires/README.md:44]: A `package` is something specific to Cosmian or just a tarball ?

Comment (Manuthor) [inline examples/yaos_millionaires/README.md:44]:
```suggestion
### Create an archive for Cosmian Enclave
```

File: examples/yaos_millionaires/client/main.py [hunk 2]
```diff
@@ -78,45 +99,142 @@ def richest(session: requests.Session, url: str, pk: bytes, sk: bytes) -> str:
...
-    url: str = sys.argv[1]
+def verify(url: str, pccs_url: Optional[str] = None) -> tuple[Path, bytes, bytes]:
     hostname, port = url_parse(url)
 
-    session: requests.Session = requests.Session()
-
     quote: Quote = ratls_verify_from_url(url)
+
+    if pccs_url:
+        _ = verify_quote(quote, None, pccs_url)
```
Comment (Manuthor) [inline examples/yaos_millionaires/client/main.py:117]:
```suggestion
        verify_quote(quote, None, pccs_url)
```
?

**Review by Manuthor — APPROVED** [review]

File: examples/yaos_millionaires/README.md (hunk 1 above)
Comment (grydz) [reply examples/yaos_millionaires/README.md]: Fixed in 966f5cb8099acb55b7881976b7964fb006dcf8ec.

**Pull request merged**